### DISCUSSION TITLE
Time difference between `containerd` releases and `k3s`'s adoption in its releases

### DISCUSSION THREAD

**Discussion by rishinair11** [k3s-io/k3s#9700] [Q&A]
There is a handy feature that we are expecting to use with the `ctr` tool during importing of images in an airgapped `k3s` environment, called `--label`. This helps us set a pinning label to the image we import using `ctr`, _while_ it's being imported. 

It's a nifty little feature, but it's available in `containerd` version v2.0.0-beta1 onwards.
I wanted to ask about the timeline (tentative) that `k3s` will use v2 `containerd`, or in fact, whats the time difference between containerd releases and `k3s`'s adoption in its releases? 

Thank you for all the work that the `k3s` team has put into this product. It's truly remarkable :)

**Comment by brandond** [comment]
Unless forced by security vulnerabilities, we don't have a hard timeline for adopting new versions of bundled components. We will probably take a look at it once the 2.0 release has been out for a while and we feel confident there are no major issues or breaking changes.
